tests/cypress/e2e/pickers/constraint.cy.ts

```diff
@@ -51,9 +51,11 @@ describe('Picker tests - Constraints', () => {
         const contentEditor = jcontent.createContent('employee');
         const pickerField = contentEditor.getPickerField('qant:employee_supervisor');
         const picker = pickerField.open();
+        cy.wait(200);
```

Comment:
Do we really need a wait here?

Reply:
from PDF test it seems we don't need
```
const picker = contentEditor.getPickerField('jdmix:fileAttachment_pdfVersion').open();
        picker.getAccordionItem('picker-media').expandTreeItem('images');
```

Reply:
I do need a wait but picker.wait() works fine, so I replaced it